src/main/java/tools/map/xml/creator/MapXmlHelper.java

```diff
@@ -1486,4 +1381,19 @@ public static String getHtmlStringFromCanalDefinitions() {
     return sb.toString();
   }
 
+  static class Wrapper<T> {
```

Comment:
This only wraps a `GameStep`, should this class really be generic? A comment indicating why it is generic and not specific to `GameStep` could be useful context for the next reader.